### PR TITLE
[DVCSMP-3311] - Add fingerprint for Sengled BR30 to Zigbee dimmer DTH

### PR DESCRIPTION
**Pull request by mikesip**

### CODE REVIEW

**Comment by tylerlange** [comment]
tested, approved to merge

**Comment by tylerlange** [comment]
@workingmonk to support br30 sengled bulb that has our logo

File: devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy [hunk 1]
```diff
@@ -30,7 +30,8 @@ metadata {
...
         fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, FC0F", outClusters: "0019", manufacturer: "OSRAM", model: "LIGHTIFY PAR38 ON/OFF/DIM", deviceJoinName: "SYLVANIA Smart PAR38 Soft White"
         fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B04, FC0F", outClusters: "0019", manufacturer: "OSRAM", model: "LIGHTIFY BR ON/OFF/DIM", deviceJoinName: "SYLVANIA Smart BR30 Soft White"
-        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E11-G13", deviceJoinName: "Sengled Element Classic"
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E11-G13", deviceJoinName: "Sengled Element Classic A19"
+	fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E12-N14", deviceJoinName: "Sengled Element Classic BR30"
```
Comment (workingmonk) [inline devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: not sure what is going on with the spacing issue here. can you fix that in the code to match the other lines

Comment (tylerlange) [reply devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: @mikesip

**Comment by workingmonk** [comment]
@tylerlange can you add a ticket number to the PR title?
if this needs QA can you assign that ticket to the QA? If not, then close that ticket

**Comment by tylerlange** [comment]
@workingmonk done

**Pull request merged**